modules/config.ts

```diff
@@ -0,0 +1,4 @@
+export const ApiServerRPCEndpoint: string = process.env.NEXT_PUBLIC_APISERVER_RPC_ENDPOINT || "http://127.0.0.1:7777"
+export const ApiServerRPCSendJSON: boolean = (String(process.env.NEXT_PUBLIC_APISERVER_RPC_SENDJSON).toLowerCase() === "true") || true
+export const PrivyAppID: string = process.env.NEXT_PUBLIC_PRIVY_APP_ID || "clz1hy6f302wbx56o75ry96nm"
+export const PrivyClientID: string = process.env.NEXT_PUBLIC_PRIVY_CLIENT_ID || "client-WY2onuhfaV4hp8h313qp56xF4BiY6tF3bh9idyWCASLct"
```

Comment:
This is somehow completely retarded but the Privy SDK does not accept random strings apparently. It's super odd. Those credentials are from a "CI" app that we created specifically for this purpose and those credentials are not used for production or anything else. So fuck my life I guess.

Reply:
Hashtag Javascript